### PR TITLE
Dec releasenotes updates

### PR DESCRIPTION
**Pull request by georgiana-caras**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 1]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
 |  | APIGOV-29228 | **Issue**: The mock server errors with 404 on APIs having a "servers" section.<br/>**Resolution**: If an API spec contains a "servers" section under a path, then the mock server will no longer respond with a 404. The "servers" section of the loaded API spec is now removed before being passed to the prism library.|
 | 01609369 | APIGOV-28448 | **Issue**: The metric for services discovered by the Discovery Agent is reflected inappropriately as "Registered Services" in the *Overview dashboard* when it should be under "Discovered Services".<br/>**Resolution**: A single metric is now displayed for Registered Services during the selected time period, which will cover both manually registered and automatically discovered services.|
+|  | APIGOV-29391 | **Issue**: API calls occasionally fail contacting API Manager.<br/>**Resolution**: Automatic retries for all API calls to v7 was added to the discovery agent. The fix is available in Axway API Management discovery agent **v1.2.11**. |
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:59]:
```suggestion
|  | APIGOV-29391 | **Issue**: API calls occasionally fail contacting API Manager.<br/>**Resolution**: Automatic retries for all API calls to v7 was added to the Discovery Agent. The fix is available in Axway API Management Discovery Agent **v1.2.11**. |
```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 2]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+  The bulk **Delete** allows you to remove multiple products at once, simplifying cleanup of your product list. An error message will appear if the delete action is not permitted, clearly stating the reason for the restriction (e.g., product not being in archive state).
+  
 ## December 4, 2024
 
 ### Agents bug fixes for December 4, 2024
 
 | Case ID | Internal ID | Description |
 |-------------|--------------|---------------------------------------------------|
-|  | APIGOV-29292 | **Issue**: Environments that have an agent without a status.version are throwing an error.<br/>**Resolution**: The error is no longer displayed in *environment details*. |
+|  | APIGOV-29292 | **Issue**: Environments that have an agent without a *status.version* are throwing an error.<br/>**Resolution**: The error is no longer displayed in *environment details*. |
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:56]:
```suggestion
|  | APIGOV-29292 | **Issue**: Environments that have an agent without a `status.version` are throwing an error.<br/>**Resolution**: The error is no longer displayed in *environment details*. |
```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 3]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+| Case ID | Internal ID | Description |
+|-------------|--------------|---------------------------------------------------|
+|  | APIGOV-29342 | **Issue**: V7 Discovery Agent credential request definitions with null identifiers.<br/>**Resolution**: Credential Request Definitions are no longer created with null indentifiers, which no longer breaks the consumer credential request flows in the Marketplace. The fix is available in Axway API Management discovery agent **v1.2.12**. |
+
+### Marketplace updates for December 12, 2024
+
+* **Custom Consumption Units Support**
+
+  (PRODUCT FOUNDRY, NEW FEATURE, PROVIDER EXPERIECE)<br />
+  We are thrilled to introduce the **Custom Consumption Units**, enabling you to define and manage custom consumption metrics for more tailored monetization strategies that better aling with the unique requirements of your offerings. You can incorporate these custom-defined units into your plan for quota allocation to control how many units a consumer is entitled to use, and monitor the usage in Business Insights.<br />
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:35]:
```suggestion
Enables you to define and manage custom consumption metrics for more tailored monetization strategies that better align with the unique requirements of your offerings. You can incorporate these custom-defined units into your plan for quota allocation to control how many units a consumer is entitled to use, and monitor the usage in Business Insights.<br />
```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 4]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+
+  **How it works**
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:37]:
```suggestion

```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 5]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+   * Define your custom unit (e.g, AI Token, Messages)
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:38]:
```suggestion

```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 6]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+   * Assign the unit to specific product plan quota
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:39]:
```suggestion

```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 7]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+   * Track and report consumption usage on this custom defined unit. This can be achieved by implementing a Custom Unit Service that will communicate with the Discovery and Traceability Agents. Refer to [Reporting custom unit usage](/docs/connect_manage_environ/connected_agent_common_reference/custom-unit-metrics) for more information about its implementation.
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:40]:
```suggestion
See [Reporting custom unit usage](/docs/connect_manage_environ/connected_agent_common_reference/custom-unit-metrics) for more information about its implementation.
```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 8]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+
+  This feature empowers you to align your quota system with custom metrics that reflect the value of your services, providing greater flexibility and revenue potential. Refer to [Manage consumption units](/docs/manage_product_foundry/manage_consumption_units) for more details.
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:42]: I don't know why line 42 is here

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 9]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+  
+* **Bulk Archive and Bulk Delete in Product Foundry List View**
+
+  (PRODUCT FOUNDRY, ENHANCEMENT)<br />
+  The bulk **Archive** functionality in the Product Foundry list view allows you to archive products in a single action.
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:47]:
```suggestion
  The bulk **Archive** functionality in the *Product Foundry list view* allows you to archive products in a single action.
```

File: content/en/docs/amplify_relnotes/2024dec_amplify.md [hunk 10]
```diff
@@ -10,15 +10,53 @@ We work hard to improve the Amplify Enterprise Marketplace experience by releasi
...
+  The bulk **Delete** allows you to remove multiple products at once, simplifying cleanup of your product list. An error message will appear if the delete action is not permitted, clearly stating the reason for the restriction (e.g., product not being in archive state).
```
Comment (lbadenhop) [inline content/en/docs/amplify_relnotes/2024dec_amplify.md:48]:
```suggestion
  The bulk **Delete** functionality allows you to remove multiple products at once, simplifying cleanup of your product list. 
```

**Comment by lbadenhop** [comment]
@georgiana-caras, I left review comments simplifying the descriptions of the enhancements. IMO, details of how the feature works should be in the docs, not the release notes. 
Also, the PR preview will not work until the markdown errors are fixed.

File: content/en/docs/amplify_relnotes/2024dec_amplify.md (hunk 9 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/2024dec_amplify.md:47]: ... archive active products in a single action

File: content/en/docs/amplify_relnotes/2024dec_amplify.md (hunk 10 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/2024dec_amplify.md:48]: ... remove multiple archived products at once....

**Review by robertengaxway — APPROVED** [review]
Made 2 suggestion for the state of the product for the bulk actions to work

File: content/en/docs/amplify_relnotes/2024dec_amplify.md (hunk 8 above)
Comment (georgiana-caras) [reply content/en/docs/amplify_relnotes/2024dec_amplify.md:39]: what is not clear about line 42 ?

Comment (lbadenhop) [reply content/en/docs/amplify_relnotes/2024dec_amplify.md:39]: There is a feature summary/description above **How it works** and then you add to the summary/description on line 42. If line 42 is necessary, it should be combined with the above summary/description. I think this is pretty detailed and lengthy for the release notes.

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 9c1e481 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-580.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by cbordier — APPROVED** [review]
I approve

**Pull request merged**